src/AzureStack/CommerceAdmin/Commerce.Admin/generate.ps1

```diff
@@ -0,0 +1,11 @@
+# Remove old

```

Comment:
While this is okay, please create an action item at your end to set the csharp-sdk-folder in the configuration file itself

Reply:
Okay.